src/main/java/echoServer/HTTP/StatusCode.java

```diff
@@ -0,0 +1,5 @@
+package echoServer.HTTP;
+
+public enum StatusCode {
+    PAGE_NOT_FOUND
```

Comment:
It makes more sense for the logic connecting a status code (like 404) to a reason ("Not Found") in this enum than separately in `HTTPResponse`. Remember Single Responsibility Principle!

This guide may be useful for that: https://www.baeldung.com/java-enum-values